core/http-auth/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~  http://aws.amazon.com/apache2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.amazon.awssdk</groupId>
+        <artifactId>core</artifactId>
+        <version>2.20.47-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>http-auth</artifactId>
+    <name>AWS Java SDK :: HTTP Auth</name>
+    <description>
+        The AWS SDK for Java - HTTP Auth module contains interfaces and implementations for http authentication
```

Comment:
nit: maybe this should use the term "generic", i.e., for generic HTTP services. Also, in these sentences when using http, maybe it should be HTTP.

Reply:
Yeah, I figured we could add the doc-comments in the follow-up PR for implementation